pkg/topolvm-node/cmd/run.go

```diff
@@ -200,3 +224,98 @@ func loadConfFile(ctx context.Context, cfgFilePath string) error {
 	)
 	return nil
 }
+
+// Struct for JSON patch operations
```

Comment:
I guess you copied these code from [aws-ebs-csi-driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver) project. Am I right?

In that case, plese add following information as a comment at the top of every function you copied.

- Where the function is copied from.
- The fact that you have modified the copied function.
- (If exists) The contents of the NOTICE file in the source project.
  - In case of EBS, add "Copyright 2022 The Kubernetes Authors."
  - ref: https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/5a56f6672dc474604bb90d65a3fc067560cf6c1b/NOTICE#L2

Example: 
https://github.com/topolvm/topolvm/blob/1b19381babdddd94a3f96cd0b2bed861ffead95f/filesystem/util.go#L189-L192
(In this case, the code is copied and not modified.)

Reply:
Sorry, but I've changed my mind.
Could you create a new file and put all copied functions in the new file?
Then, add a comment at the top of the file.